OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan_Optimized dashboard for mobile view

# Description
ALL THE RED LINES FIXED
![Screenshot 2024-08-24 130520](https://github.com/user-attachments/assets/e19af1f7-7e7f-42cc-ba7f-71c30a1d986c)

## Related PRS (if any):
This frontend PR is related to the dev backend

## Main changes explained:
- Aligned dashboard components when 991px and below
- Created new dropdown for days sector when below 768px.
- Aligned Tracking/Hide button to the right

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard
6. verify dashboard components are aligned when 991px and below
7. verify that days selector displays correctly when 768px and below
8. verify that Tracking/Hide button is aligned to the right.
9. verify these new features works in [dark mode](https://docs.google.com/document/d/11OXJfBBedK6vV-XvqWR8A9lOH0BsfnaHx01h1NZZXfI)

## Screenshots or videos of changes:
![Screenshot 2024-08-24 115005](https://github.com/user-attachments/assets/41e5c924-6e67-4c77-9e87-432b65ad5fe4)
![Screenshot 2024-08-24 130151](https://github.com/user-attachments/assets/ce7bb29a-054a-433d-b813-1e5295063bc6)
